### PR TITLE
Added Cybersuit Corp set (was Corpo World)

### LINKED ISSUE #2664: Ask to create a testing site for new set: BM Corpo World.
Hi,
Please kindly create a test site for testing newly proposed BM set.

Buttons in the set include:
Andrew ks(6) (8) fI(12) (X) (X)
Chad ks(4) (4) It(12) (20) (X)
Fiona (6) fM(10) st(12) (12) (X)
Gavin  (6) fM(8) Is(12) (20) (X)
Isabel  (6) It(8) fs(10) (12) (X)
Monique  kt(8) (8) fs(12) (12) (X)
Naima  (4) st(10) fI(10) (12) (X)
Sachin (6) kt(6) Is(10) (20) (X)

Flavor text - empty for now
Art - empty for now

Many thanks
AnnoDomini

### PR DESCRIPTION
**Pull request by blackshadowshade**
I've updated this pull request to be mergeable once we are happy with it. Fixes #2664.

~This is the initial testing pull request addressing #2664. @cgolubi1, please deploy a test site for this when you can, so that testing can begin.~

~This is NOT TO BE MERGED, and I'm deliberately allowing the unit tests to fail until testing is completed.~

### CODE REVIEW

**Comment by cgolubi1** [comment]
Here you go - http://temporary.dev.buttonweavers.com/ui/buttons.html?set=Corpo%20World

**Comment by cgolubi1** [comment]
I'm setting up a replay site too, and i'll just have it play a bunch of novel Corpo World vs any games to make sure we don't have any skill combos in these buttons that cause crashes or stuck games.

**Comment by cgolubi1** [comment]
Replay testing played something like 3400 games, which should have been enough to catch obvious crash bugs over such a small number of buttons.  Calling that good.

**Comment by TheOrgg** [comment]
The test site seems like it is messed up at the moment.  I also posted this on the Issue when I thought it was just me.  Anno has trouble with it, too.

**Comment by cgolubi1** [comment]
Looks like mysqld got OOM-killed:

```
[Sun Jan 31 06:53:29 2021] Out of memory: Kill process 21828 (mysqld) score 303 or sacrifice child
[Sun Jan 31 06:53:29 2021] Killed process 21828 (mysqld) total-vm:1121964kB, anon-rss:182280kB, file-rss:0kB
```

**Comment by cgolubi1** [comment]
I don't have an explanation for why that happened.  It should be back online now.

**Comment by blackshadowshade** [comment]
I've added a new database update for the requested changes.

**Comment by cgolubi1** [comment]
Sorry to be so slow to update this; i missed the February commit.  Working on it now.

**Comment by cgolubi1** [comment]
Site updated; sorry to be slow.

**Comment by blackshadowshade** [comment]
Sorry, left a message in #2664, but meant it to be here:

@cgolubi1, I've added an update file that adds these new testing recipes to the database as Fiona1, Nadia1, Gavin1, Fiona2, Nadia2, Gavin2.

I haven't changed data.button.sql, I'll clean all this up when testing is done and I recreate the files from scratch.

**Comment by TheOrgg** [comment]
These are mostly finished, with one or two more characters to test, and I'm uploading them for putting into the test site to take a look at it and see what needs adjusting, if anything, to the graphical elements of the buttons.

150GavinBlank
150pxAndrew
150pxChad
150pxFiona
150pxIsabel
150pxMonique
150pxNadia
150pxSachin

**Comment by TheOrgg** [comment]
Final 150px images for the set when it goes live.

150pxSachin
150pxAndrew
150pxChad
150pxFiona
150pxGavin
150pxIsabel
150pxMonique
150pxNadia

**Comment by blackshadowshade** [comment]
Argh, no 1337!  XD

**Comment by TheOrgg** [comment]
That's up to A_D, ain't it?  Haven't heard back since the last email, but he did confirm that he's been really busy.

**Comment by blackshadowshade** [comment]
Any confirmation of the set name yet?

**Comment by TheOrgg** [comment]
Not yet.  Just set up a game to get confirmation.

**Comment by TheOrgg** [comment]
Anno wants non-leet CyberSuit Corp.

**Comment by blackshadowshade** [comment]
This pull request should be ready for final review. It will conflict with any other pull request that changes the number of playable buttons or button sets.

**Comment by TheOrgg** [comment]
So CyberSuit will be included with the giant set update!  Hooray!

**Comment by blackshadowshade** [comment]
I've added the flavour text in an additional database update, and also updated data.button.sql.

No rebase yet, so that we can easily update the testing site.

**Comment by blackshadowshade** [comment]
This has been rebased to resolve conflicts created by merging 2021 Fanatics.

**Comment by cgolubi1** [comment]
So, there was a dev site for this, but AWS stopped it awhile ago.  I could:
* reboot it, update the DNS, and apply the new DB update
* get rid of it and stand up a new dev site

The second of those is probably slightly easier, but neither is particularly hard.  Anyone have a preference?

**Comment by TheOrgg** [comment]
Are they not ready to go to the Live site?

File: deploy/database/updates/02664_corpo_world_01.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+INSERT INTO buttonset (id, name, sort_order) VALUES
+(86, 'Corpo World', 8600);
```
Comment (cgolubi1) [inline deploy/database/updates/02664_corpo_world_01.sql:2]: This is wrong --- it still has the old set name.

**Comment by cgolubi1** [comment]
DB update-vs-rebuild test failed --- see inline comment.  Fix that, and let me know if you (Shadowshade) are good with this being merged once that test passes.

**Comment by blackshadowshade** [comment]
I've fixed the update file, that should now work.

I believe that we haven't run any replay testing on this set, so I think that's still waiting to happen.

**Comment by cgolubi1** [comment]
Sounds good, will do.  I don't have any specific skill interaction concerns with this set, but if i don't have to guess, i don't have to guess wrong.

**Comment by cgolubi1** [comment]
Verified that DB updates are fine now.

**Comment by cgolubi1** [comment]
Okay, this has played 7200 new games since yesterday, and hasn't crashed.  Good enough?  Should we merge it?  If not, remind me what else has to happen.

**Comment by blackshadowshade** [comment]
I'm good to merge this, then I'll update the other pull requests to deal with the resulting merge conflicts.

**Pull request merged**